src/common/featuremanager/FeatureManagerService.js

```diff
@@ -986,6 +986,10 @@
           if (goog.isDefAndNotNull(source.getGetFeatureInfoUrl)) {
             validRequestCount++;
           }
+          // include special case for search layer and results
+          else if (layer.get('metadata').searchLayer || layer.get('metadata').searchResults) {
+            validRequestCount++;
+          }
```

Comment:
Adjusting this not to hijack the getFeatureInfo workflow, but rather operate alongside it (still within the service's function for giving features back based on the click event, but not jammed into this block of the code)